src/event.js

```diff
@@ -424,8 +426,11 @@ jQuery.event = {
 			delegateCount = handlers.delegateCount,
 			cur = event.target;
 
+		// Support (at least): Chrome, IE9
```

Comment:
Well, the support comment as is already makes it confusing as to which parts of the conditional apply to which bug. I'd suggest making that clear and then testing the specific workarounds in the supported browsers to get accurate support comments.